cli/src/pcluster/resources/imagebuilder/parallelcluster_test.yaml

```diff
@@ -15,6 +15,30 @@ phases:
               NVSWITCHES=$(lspci -d 10de:1af1 | wc -l)
               echo "${NVSWITCHES}"
 
+      - name: OperatingSystemName
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              FILE=/etc/os-release
+              if [ -e ${FILE} ]; then
+                . ${FILE}
+                RELEASE="${ID}${VERSION_ID:+.${VERSION_ID}}"
```

Comment:
I suppose these variables are defined in `/etc/os-release`, right?

Reply:
yep
```
$ cat /etc/os-release
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
```